src/cdk/utils/create-lambda-integration.ts

```diff
@@ -50,6 +56,23 @@ export function createLambdaIntegration(
     }
   );
 
+  const secretsManagerPolicy = new aws_iam.PolicyStatement({
```

Comment:
Please rename to `secretsManagerPolicyStatement` or use it inline in line no. 71

Reply:
check again